### ISSUE TITLE
Documentation - Installalation

### ISSUE THREAD

**Issue by aaronemassey** [emacs-lsp/lsp-mode#3044]
tldr: Make it clear that dash.el is required.

Something really minor but would likely help newbies to the Emacs ecosystem is to clearly state in the installation (most preferred), troubleshooting, FAQ, or some other location that dash.el is a required Elisp dependency. Otherwise "-compose" comes up as symbol void. I have not been able to find such documentation besides GitHub issues.

Sure, dash.el as a requirement can be discovered while looking through the source and GitHub issues, but the immediate connection might not be made by someone relatively new to the Emacs ecosystem (let alone Emacs itself). To me, if I'm having to dig through GitHub issues to discover how to do an initial setup, something is not quite right.

**Comment by ericdallo** [comment]
I don't understand the issue, if you install lsp-mode properly and don't have dash.el installed, it will install dash when install lsp-mode, right?

**Comment by aaronemassey** [comment]
This was not the case for me. That's interesting. Ill take a fresh setup and try to reproduce. To give a list of steps.

If what you say is true then that'd make sense. Maybe this is an issue for something else then. Thank you.
